README.md

```diff
@@ -1,4 +1,5 @@
 # secure-electron-context-menu
+
```

Comment:
Can you remove these new lines? Syntactically I know it doesn't make a difference, but I'd like to keep the headers as they were before in the readme.